data/json/npcs/robofac/intercom/robofac_intercom_prototypes.json

```diff
@@ -257,7 +257,7 @@
     "id": "TALK_ROBOFAC_INTERCOM_ARMOR_BREAKTHROUGH",
     "//": "armor reward blurb after waiting",
     "type": "talk_topic",
-    "dynamic_line": "Yes, we've made great progress!  I'll spare you the details but we've developed much more resilient materials for our armors while sacrificing no maneuverability.  As a thank you for bringing this to us the first working model is yours merc.  That said these do take a considerable amount of high-end materials.  If you want the matching pieces we'll need 6 of our coins to barely cover the materials cost.  Also, I've been told to inform you we would be happy to collaborate like this again in the future.",
+    "dynamic_line": "Yes, we've made great progress!  The data on the template was invaluable.  I'll spare you the details, it allowed us to produce much more resilient materials for the attachments without sacrificing the agility.  Testing is still ongoing, but-\"  The voice quickly confirms something off-mic.  \"We are willing to provide you with the first working model, but these prints take a considerable amount of high-end materials.  If you want the matching pieces, we would like a flat fee of 6 HGC to cover the materials cost.\"\n\n\"I've also been told to inform you that we would be happy to collaborate like this again in the future.",
```

Comment:
I don't think "the agility" sounds less awkward than maneuverability here. It also IMO sounds less like someone with modern combat training.